"build:dev" script fails

## Expected behavior

`yarn run build:dev` should build successfully.

## Actual behavior

`yarn run build:dev`fails with error:
```
ERROR in src/app/containers/check-in/check-in.component.ts(7,1): error TS1202: Import assignment cannot be used when targeting ECMAScript modules. Consider using 'import * as ns from "mod"', 'import {a} from "mod"', 'import d from "mod"', or another module format instead.
```
 [Travis build](https://travis-ci.org/cerberus-org/cerberus-app/builds/335848375)

## Steps to reproduce

1. Run command `yarn run build:dev`.

--- a/.travis.yml
+++ b/.travis.yml
@@ -34,6 +34,7 @@ install:
 script:
   - yarn run test:ci
   - yarn run lint
+  - yarn run build:dev
 
 before_deploy:
   - if [[ "$TRAVIS_BRANCH" = "master" ]]; then yarn run build:dev; fi

--- a/src/app/containers/check-in/check-in.component.ts
+++ b/src/app/containers/check-in/check-in.component.ts
@@ -2,10 +2,10 @@ import { Component, OnDestroy, OnInit, ViewChild } from '@angular/core';
 import { MatTabGroup } from '@angular/material';
 import { ActivatedRoute } from '@angular/router';
 import { Store } from '@ngrx/store';
+import * as _ from 'lodash';
+import 'rxjs/add/operator/distinctUntilChanged';
 import { Subscription } from 'rxjs/Subscription';
 
-import _ = require('lodash');
-import 'rxjs/add/operator/distinctUntilChanged';
 import * as AppActions from '../../actions/app.actions';
 import * as CheckInActions from '../../actions/check-in.actions';
 import { HeaderOptions } from '../../models/header-options';

